pages/blog/2022Q1-summary.md

```diff
@@ -0,0 +1,175 @@
+
+- A week ago, we stopped using our `asyncapi-users` Google Group. We removed it, and all the members were migrated as subscribers of the AsyncAPI Newsletter. This is the place where we will now send email notifications every Monday-morning with a list of upcoming meetings,
+- We switched to the new Google Calendar instance managed by the official AsyncAPI Google account. You should remove/unsubscribe from the old calendar and [subscribe to the new one](https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t),
+- [Our Technical Steering Committee (TSC)](https://github.com/asyncapi/community/discussions/295) approved to sponsor 8 Zoom licenses to host official AsyncAPI meetings. So far, only 3 licenses have been assigned. We have lots of room for more meetings and hosts, if there is a need.
+
+Huge thanks to [Krishna Kumar](https://github.com/Krishks369) and [Shreyansh Jain](https://github.com/sudoshreyansh) that helped in research related to the implementation :heart:
+
+If you want to learn more about becoming a host of the meeting, or other things related to meetings, get familiar with our [dedicated meetings FAQ](https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md).
+
+Oh, and I almost forgot, look at this :point_down: :star: 
+All [info about AsyncAPI-related meetings](https://www.asyncapi.com/community/meetings) in one beatifull place :heart:
+
+<Figure
+  src="/img/posts/2022Q1-summary/meetings.webp"
+  className="text-center"
+  caption="View of dedicated community/meetings page on the AsyncAPI website."
+/>
+
+## Upcoming 2.4 and 3.0 releases of the spec
+
+We released a new [2.4.0 version of AsyncAPI Specification](https://github.com/asyncapi/spec/releases/tag/v2.4.0). There are many good features, like the possibility to provide the `id` of the Message unique for the entire AsyncAPI file or the ability to specify security valid only for a given operation. You definitely need to read [release notes](https://www.asyncapi.com/blog/release-notes-2.4.0).
+
+Huge shout out to [Sergio Moya](https://twitter.com/smoyac), who was a release coordinator for 2.4 release and pushed all the doors to make sure the release could go out.
+
+2.4.0 was released, but this doesn't block us from working on something much bigger, 3.0.0 version of the specification :rocket:
+
+If you do not know yet about it, I highly recommend you to have a look at [this article](https://www.asyncapi.com/blog/async-api-spec-3.0-release) from [Harsh Mishra](https://www.linkedin.com/in/harshcasper/).
+
+Please join these efforts, as there is much work ahead of us, but not many hands doing actual work. We meet every two weeks to sync the work efforts and discuss important topics. Feel free to join :pray:
+
+I also recommend following [Sergio Moya](https://twitter.com/smoyac) and reaching for [his regular updates on specification-related efforts](https://gist.github.com/smoya)
```

Comment:
```suggestion
I also recommend following [Sergio Moya](https://twitter.com/smoyac) and reaching for his [regular updates on specification-related efforts](https://gist.github.com/smoya)
```
I moved `his` outside of the anchor text because if a screen reader user is "scanning" only the links (they have a feature that allows them to click link by link), I don't think that it will be clear what's the **antecedent** of the pronoun **his**. So I think that a screen reader will feel it's clearer to have the link only say _"regular updates on specification-related efforts"_.

CAVEAT: The BEST anchor text for that situation would be to include `Sergio` vs `his`, but you will probably feel it sounds too repetitive to read the name Sergio a second time in this sentence. This is why I thought it was better to just keep the pronoun "his" outside of the anchor text. The anchor text is still very specific and clear, even without including the name of the author updates.